<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatMainWindow</class>
 <widget class="QMainWindow" name="ChatMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Disease Prediction</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="predictButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>240</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Sends a message to other people</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unSendButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Sends a message to other people</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>Unsend</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="diseaseHistory">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>Messages sent and received from other users</string>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>210</y>
      <width>581</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Sends a message to other people</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Symtomps</string>
    </property>
    <property name="buddy">
     <cstring>messageLineEdit</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Disease</string>
    </property>
    <property name="buddy">
     <cstring>messageLineEdit</cstring>
    </property>
   </widget>
   <widget class="QTextBrowser" name="chatHistory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>Messages sent and received from other users</string>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>240</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Message:</string>
    </property>
    <property name="buddy">
     <cstring>messageLineEdit</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="messageLineEdit">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>240</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Sends a message to other people</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <zorder>chatHistory</zorder>
   <zorder>messageLineEdit</zorder>
   <zorder>sendButton</zorder>
   <zorder>label</zorder>
   <zorder>predictButton</zorder>
   <zorder>unSendButton</zorder>
   <zorder>diseaseHistory</zorder>
   <zorder>clearButton</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionChangeNickname">
   <property name="text">
    <string>Change nickname...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>chatHistory</tabstop>
  <tabstop>messageLineEdit</tabstop>
  <tabstop>sendButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered(bool)</signal>
   <receiver>ChatMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messageLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>sendButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
